CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## Version 2.0.1 (Under development)
 
-### App Center Data
+### AppCenterData
 
 * **[Fix]** Fix an issue where invalid characters in the document ID are accepted at creation time but causing errors while trying to read or delete the document. The characters are `#`, `\`, `/`, `?`, and all whitespaces.
 
+### AppCenterDistribute
+
+* **[Fix]** Fix a rare crash on downloading the update.
```

Comment:
I think you can give a bit more details here instead of "a rare crash".

Reply:
Yes since you have a repro now that can explain the crash, please update the description.